internal/kernel_neon.h

```diff
@@ -727,7 +985,8 @@ struct NEON_64bit_GEMM_Int8Operands_LhsNonzero : KernelBase {
         "beq " GEMMLOWP_LABEL_AFTER_LOOP_LAST16 "f\n"
 
         // General loop.
-        GEMMLOWP_LABEL_LOOP ":\n"
```

Comment:
Minor not - Are these (and the similar ones below) significant changes, or just formatting diffs? If they're just formatting, would it be possible to revert them to keep the file history a bit less cluttered?